wps-scripts/src/main/resources/archetype-resources/src/main/java/WpsScriptsPackage.java

```diff
@@ -0,0 +1,316 @@
+package ${package};
+
+import org.apache.commons.io.IOUtils;
+import org.orbisgis.wpsservice.LocalWpsService;
+
+import org.orbisgis.wpsservice.controller.process.ProcessIdentifier;
+import org.orbisgis.wpsclient.WpsClient;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+/**
+ * Main class of the plugin which declares the scripts to add, their locations in the process tree and the icons
+ * associated.
+ * In the WpsService, the script are organized in a tree, which has the WpsService as root.
+ *
+ * Scripts can be add to the tree under a specific node path with custom icon with the following method :
+ *      localWpsService.addLocalScript('processFile', 'icon', 'boolean', 'nodePath');
+ * with the folowing parameter :
+ *      processFile : The File object corresponding to the script. Be careful, the plugin resource files can't be
+ *              accessed from the outside of the plugin. So you have to copy it (in a temporary file as example) before
+ *              adding it to the WpsService.
+ *      icon : Array of Icon object to use for the WpsClient tree containing the processes. The first icon will be used
+ *              for the first node of the path, the second icon for the second node ... If the node already exists,
+ *              its icon won't be changed. If there is less icon than node, the last icon will be used for the others.
+ *              If no icon are specified, the default one from the WpsClient will be used.
+ *      boolean : it SHOULD be true. Else the used will be able to remove the process from the WpsClient without
+ *              deactivating the plugin.
+ *      nodePath : Path to the node where the process should be add. If nodes of the path doesn't exists, they will be
```

Comment:
In the WpsClient, the processes are organized under a tree which contains nodes, subnodes, subsubnodes .... A process is a node, a folder is a node ... 
So in this archetype, the user can set under which nodes the process should be add.